lte/gateway/c/core/oai/tasks/s1ap/s1ap_state_converter.cpp

```diff
@@ -60,6 +60,14 @@ void S1apStateConverter::state_to_proto(s1ap_state_t* state, S1apState* proto) {
     FREE_HASHTABLE_KEY_ARRAY(keys);
   }
 
+  keys                        = hashtable_ts_get_keys(&state->enbs);
```

Comment:
This looks to have slipped from the other PR for protobuf state eNB sync, is this PR dependent on the other one?

Reply:
Hi Alex, I am not planning to merge this change instead I will merge the sub commits, this change is just to give you an overall view of where I am going with things.